plugins/analyser_plugin.py

```diff
@@ -17,6 +17,7 @@ def print_all(blocks: List[Element], indent="    ┃       "):
                 if isinstance(block, List):
                     indent += "    "
                     print_all(block, indent)
+                    indent = indent[:len(indent) - len("    ")]
```

Comment:
after "exiting" the list, remove the indentation

Reply:
Makese sense!